apps/dapp-console-api/docker-compose.yml

```diff
@@ -22,6 +33,8 @@ services:
     env_file: .env
     environment:
       - REDIS_URL=redis://redis:6379
+      - DB_HOST=postgres
+      - DB_PORT=5432
```

Comment:
it might be worth setting all of the db env vars here instead of in the env file

Reply:
yeah, I like that.